Isosurfacing_3/doc/Isosurfacing_3/Isosurfacing_3.txt

```diff
@@ -0,0 +1,327 @@
+namespace CGAL {
+/*!
+
+\mainpage User Manual
+\anchor Chapter_Isosurfacing3
+
+\cgalAutoToc
+\author Mael Rouxel-Labbé, Julian Stahl, Daniel Zint, and Pierre Alliez
+
+\cgalFigureAnchor{IsosurfacingTeaser}
+<center>
+<img src="isosurfacing_teaser.png" style="max-width:50%;"/>
+</center>
+\cgalFigureCaptionBegin{IsosurfacingTeaser}
+Generating a surfacing from a 3D gray level image using Marching Cubes (3D input image: qim.dk)
+\cgalFigureCaptionEnd
+
+\section SecIsoSurfacingIntroduction Introduction
+
+Given a scalar field, an isosurface is defined as the locus of points where the scalar field
+has a given constant value; in other words, it is a level set.
+This constant value is referred to as "isovalue", and, for well-behaved scalar fields,
+the level set forms a surface.
+"Isosurfacing", also known as "isosurface extraction" or "contouring", is the process of constructing
+the isosurface corresponding to a given scalar field and isovalue.
+This process is often needed for volume visualization and the simulation of physical phenomena.
+
+This \cgal package provides methods to extract isosurfaces from 3D scalar fields.
+These contouring techniques rely on the discretization of the 3D space to construct
+an approximate representation of the isosurface.
+The 3D scalar field can be described through various representations: an implicit function,
+an interpolated set of discrete sampling values, etc. (see \ref SecIsosurfacingExamples). The isovalue is user-defined.
+The output is a polygon soup, made either of triangles or quads depending on the method,
+and can be a single connected component, or made of multiple, disjoint components.
+Note that due to the inherent approximate nature of these discrete methods, parts of the "true"
+isosurface may be missing from the output, and the output may contain artifacts that are not present
+in the true isosurface.
+
+\section SecIsosurfacingMethods Isosurfacing Methods
+
+The scientific literature abounds with algorithms for extracting isosurfaces, each coming
+with different properties for the output and requirements for the input \cgalCite{cgal:dljjaw-sisp-15}.
+This package offers the following methods
+<ul>
+ <li>Marching Cubes \cgalCite{LC87}</li>: a simple and efficient method that generates a triangle mesh, with almost no guarantees.
+ <li>Topologically Correct Marching Cubes \cgalCite{cgal:g-ctcmi-16}</li>: an extension to Marching Cubes that provides additional guarantees for the output.
+ <li>%Dual Contouring \cgalCite{cgal:jlsw-dchd-02}</li>: a method that generates a polygon mesh, with focus on sharp features recovery.
+</ul>
+
+\subsection SubSecMarchingCubes Marching Cubes (MC)
+
+Marching Cubes (MC) uses a volumetric grid, i.e., a 3D iso-cuboid partitioned into hexahedral cells.
```

Comment:
After i.e.   is always a comme.    Why would you like to make it italics?

Reply:
Because I thought it was not correct to have point+comma but it seems I'm wrong.

Italics because foreign languages should always be in italics? It seems I'm also wrong and this concerns only non-familiar words.

So let me resolve this discussion